specutils/fitting/fitmodels.py

```diff
@@ -0,0 +1,507 @@
+from __future__ import division
+
+import functools
+import operator
+
+import numpy as np
+import astropy.units as u
+
+from ..spectra.spectrum1d import Spectrum1D
+from ..manipulation.utils import excise_regions
+from astropy.modeling import fitting, Model, models
+
+
+__all__ = ['fit_lines']
+
+
+def fit_lines(spectrum, model, fitter=fitting.SimplexLSQFitter(),
+              exclude_regions=None, weights=None, window=None):
+    """
+    Fit the input models (initial conditions) to the spectrum.  Output will be
```

Comment:
What about replacing the first sentence with "Fit the input models to the spectrum. The parameter values of the input models will be used as the initial conditions for the fit."

Reply:
switched to your version